### PR TITLE
Add some documentation to the action class to avoid confusion

### PR DESCRIPTION
**Pull request by duke8253**

### CODE REVIEW

**Comment by a-canary** [comment]
Would you mind throwing something in descriptions of TSActionDone and TSActionCancel
https://github.com/apache/trafficserver/blob/2d76d6421c14d343d5ff36733d74da238e9c3f22/doc/developer-guide/api/functions/TSActionDone.en.rst

**Comment by duke8253** [comment]
> Would you mind throwing something in descriptions of TSActionDone and TSActionCancel
> https://github.com/apache/trafficserver/blob/2d76d6421c14d343d5ff36733d74da238e9c3f22/doc/developer-guide/api/functions/TSActionDone.en.rst

Didn't know we had a file for that. I'll add some in there.

**Comment by duke8253** [comment]
[approve ci]

**Comment by duke8253** [comment]
[approve ci]

File: doc/developer-guide/api/functions/TSActionCancel.en.rst [hunk 1]
```diff
@@ -30,3 +30,9 @@ Synopsis
 
 Description
 ===========
+
+Cancels the action.
+
+.. note::
+    The argument ``actionp`` must be checked prior to calling this function,
```
Comment (SolidWallOfCode) [inline doc/developer-guide/api/functions/TSActionCancel.en.rst]: Use
```
:arg:`actionp`
```

**Pull request merged**